### ISSUE TITLE
Requests submitted through the Contact Form getting assigned to the first Sales Team

### ISSUE THREAD

**Issue by pawa2007** [odoo/odoo#34606]
Impacted versions: 12.0

Steps to reproduce:

1. CRM > Configure > Sales Teams: Create 2 Sales Teams - "Sales" and "Website". 

2. Website > Configuration > Settings, scroll to the bottom of the page and select "Website" as the Default Sales Team for the Contact Form.

Current behavior:
Every time a request is submitted through the website it gets auto allocated to the "Sales" team. We also observed that if we rename the "Website" sales team to "1 - Website" so that it appears first in the list view, the website requests start getting auto assigned to the "1 - Website" sales team.

Expected behavior:
Requests submitted using the Contact Form from the website should get auto-allocated to the "Website" sales team since it has been specified as the default sales team.

Video/Screenshot link (optional):

**Comment by pedrobaeza** [comment]
This is supposed to be configured here:

![Selección_023](https://user-images.githubusercontent.com/7165771/60706257-5caeea80-9f09-11e9-8740-b06bfbb0b1e1.png)

Can you please modify the value there and tell back if it's working?

**Comment by pawa2007** [comment]
That's precisely where I set the value to the "Website" sales team. The requests submitted through the Contact Form get auto-allocated to the "Sales" sales team in spite of this. I have left the "alias" blank on both Sales Teams since the email addresses have not been set up yet.

I also noticed that if I rename the teams to "1 - Website" and "Sales" so that "1 - Website" shows up first in the list, the requests start getting auto-allocated to "1 - Website" which could mean that the allocation is always happening to the first team in the list. However, it could just be a coincidence.
